cluster/charts/crossplane/templates/environment-personas-clusterroles.yaml

```diff
@@ -0,0 +1,156 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-env-admin
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-environment-admin: "true"
+rules: [] # Rules are automatically filled in by the controller manager.
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:env:default:admin
+  labels:
+    rbac.crossplane.io/aggregate-to-environment-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - stacks.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-env-edit
+  labels:
+    rbac.crossplane.io/aggregate-to-environment-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-environment-edit: "true"
+rules: [] # Rules are automatically filled in by the controller manager.
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:env:default:edit
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane-edit: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "environment"
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - stacks.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - create
```

Comment:
What about verbs like `watch`, or `list`?

Reply:
This is an interesting question.. 

The `crossplane-env-edit` role would pick up the `get`, `watch`, and `list` verbs because of https://github.com/crossplaneio/crossplane/pull/1059/files/9e5ec6336d2854c06bd962e9ad8918e6edc6ec5a#diff-de6ef6770bec0bdeca8902bda7136f60R107

In the design doc, we state that some of the cluster roles, ``stack:{repo-name}:{stack-name}:{version}:{role=system|admin|edit|view}`, are for internal use only.

If `crossplane:stack-manager:{scope}:default:{role}` cluster roles are for internal use only, then we don't need to worry about the verbs they provide, so long as `crossplane-{scope}-{role}` picks up all the needed verbs through aggregation.  However, if we expect users to consume the `crossplane:stack-manager:{scope}:default:{role}` clusterroles directly, we should include the read-only verbs.

We've been using the Kubernetes model as a guide. 
```
kubectl get -o yaml clusterrole view,system:aggregate-to-view
kubectl get -o yaml clusterrole  edit,system:aggregate-to-edit
kubectl get -o yaml clusterrole  admin,system:aggregate-to-admin
```